From f63878b239c379d2f44da671541fd5ce81940fdb Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 13 May 2024 15:03:00 +0200
Subject: [PATCH] slic3r: Fix std::regex declaration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

/run/build/BambuStudio/src/slic3r/GUI/Jobs/PrintJob.cpp: In member function ‘virtual void Slic3r::GUI::PrintJob::process()’:
/run/build/BambuStudio/src/slic3r/GUI/Jobs/PrintJob.cpp:291:22: error: ‘regex’ is not a member of ‘std’; did you mean ‘boost::regex’?
  291 |                 std::regex pattern("_+");
      |                      ^~~~~
---
 src/slic3r/GUI/Jobs/PrintJob.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/slic3r/GUI/Jobs/PrintJob.cpp b/src/slic3r/GUI/Jobs/PrintJob.cpp
index d3b642bd74d7..c1ec08945b98 100644
--- a/src/slic3r/GUI/Jobs/PrintJob.cpp
+++ b/src/slic3r/GUI/Jobs/PrintJob.cpp
@@ -1,4 +1,5 @@
 #include "PrintJob.hpp"
+#include <regex>
 #include "libslic3r/MTUtils.hpp"
 #include "libslic3r/Model.hpp"
 #include "libslic3r/PresetBundle.hpp"
-- 
2.44.0

